services/movie.py

```diff
@@ -0,0 +1,42 @@
+from django.db.models import QuerySet
+
+from db.models import Movie
+
+
+def get_movies(
+        genres_ids: list[int] = None,
+        actors_ids: list[int] = None
+) -> QuerySet:
+    if not genres_ids and not actors_ids:
+        return Movie.objects.all()
+
+    if genres_ids and actors_ids:
+        return (Movie.objects.filter(genres__in=genres_ids)
+                .filter(actors__in=actors_ids))
```

Comment:
When filtering by both genres and actors, the current implementation might not return movies that match both criteria simultaneously. Consider using `Q` objects with `&` to ensure both conditions are met for each movie.